packages/faency/src/components/InputTags.story.tsx

```diff
@@ -0,0 +1,89 @@
+import { storiesOf } from '@storybook/react'
+import React, { useState } from 'react'
+import { Box } from './Box'
+import { InputTags } from './InputTags'
+import { DismissableChip } from './DismissableChip'
+import { Text } from './Text'
+
+const InputTagsWithAutocomplete = () => {
+  const [tags, setTags] = useState(['Traefik', 'Compression'])
+  const allOptions = ['Redirect Path', 'Redirect Regex', 'Redirect Scheme']
+  const [options, setOptions] = useState(allOptions)
+  const [submitted, saveSubmit] = useState<string[]>([])
+
+  return (
+    <>
+      <InputTags
+        placeholder="Input with autocomplete"
+        tags={tags}
+        options={options}
+        onChange={v => setOptions(() => allOptions.filter(o => o.includes(v)))}
```

Comment:
On this example I would expect to see a new tag selecting an option.

Reply:
This should be the use case for the market place, where there is no correlation between tags and the input.
Because of that, I preferred to show the entered values in a separated set, do you think it's better to do that in some other way?